Jenkinsfile

```diff
@@ -1,28 +1,11 @@
-pipeline {
-    options {
-        buildDiscarder(logRotator(numToKeepStr: '20'))
-        timeout(time: 1, unit: 'HOURS')
-    }
-    agent {
-        // TODO convert as in https://github.com/jenkinsci/archetypes/pull/44
-        // (Archiving of build.log would be tricky; maybe just pass -Dinvoker.streamLogs instead.)
-        docker {
-            image 'maven:3.6.1-jdk-8'
-            label 'docker'
-        }
-    }
-    stages {
-        stage('main') {
-            steps {
-                // TODO -s settings-azure.xml
-                sh 'mvn -B --no-transfer-progress -Prun-plugin-pom-its clean verify'
-            }
-            post {
-                failure {
-                    catchError { // JENKINS-42478: in case the failure occurred prior to getting the node
-                        archiveArtifacts artifacts: 'target/its/*/build.log', allowEmptyArchive: true
-                    }
-                }
+properties([buildDiscarder(logRotator(numToKeepStr: '20'))])
+node('maven') {
+    checkout scm
+    timeout(time: 1, unit: 'HOURS') {
+        // TODO Azure mirror
+        ansiColor('xterm') {
+            withEnv(['MAVEN_OPTS=-Djansi.force=true']) {
+                sh 'mvn -B -Dstyle.color=always -ntp -Prun-plugin-pom-its clean verify'
```

Comment:
Could we use `runMaven` instead? Mirros & Co

Reply:
`runMaven` does not set up mirrors, nor has it been updated for `-ntp`, and does a bunch of stuff which is useless in an ACI agent. I prefer to keep things more transparent and explicit.

Reply:
> runMaven does not set up mirrors, nor has it been updated for -ntp, and does a bunch of stuff which is useless in an ACI agent

Hm? https://github.com/jenkins-infra/pipeline-library/blob/master/vars/infra.groovy#L69-L89

Maybe you are talking about the Pipeline Maven Plugin. Sorry, I meant `infra.runMaven`. We can add `-ntp` there and any ACI agent tweaks if needed there

Reply:
Can be done in a follow-up, though I would still prefer to call `retrieveMavenSettingsFile` directly and avoid `runMaven`: nicer to have a single command you can copy and paste into a local shell to check build failures. Excessive Groovy functions just obscures what is actually being run.